boutdata/squashoutput.py

```diff
@@ -36,6 +38,9 @@ def squashoutput(
     """
     Collect all data from BOUT.dmp.* files and create a single output file.
 
+    Note: adds an attribute to the 'squashed' output file called `squashoutput_version`
+    which records the current version number of `boutdata`.
+
```

Comment:
Not sure that is a an important enough detail to be documented here.

If if I find `squashoutput_version` in the attributes, I have a good guess where it is coming from ...

Reply:
I kinda agree. I felt it was worth a note because `squashoutput()` is intended (at least in my mind, I don't think it's documented anywhere...) to put data into 1 file 'without modifying it'; so when the script adds something extra, it's worth noting. Also, (at least if we built some docs for boutdata on readthedocs.io...) someone who just got given a data file would have some chance of finding out what this attribute was for just by googling it, and without needing to read any Python code.

@ZedThree any thoughts?

Reply:
I agree it's good to have it in the docs. As you say, makes it a bit more discoverable what it actually is.